integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/IntegrasjonspunktNokkel.java

```diff
@@ -66,18 +38,18 @@ public PrivateKey loadPrivateKey() {
         PrivateKey key = null;
         try (InputStream i = openKeyInputStream()) {
             KeyStore keystore = KeyStore.getInstance("JKS");
-            keystore.load(i, pkPassword.toCharArray());
+            keystore.load(i, properties.getOrg().getKeystore().getPassword().toCharArray());
```

Comment:
Kanskje vi burde fortsette å ha en IntegrasjonspunktConfig som en wrapper for å hente ut properties. Klasser burde ikke kjenne til så dype hierarkier inni andre klasser. Koden blir mer vanskelig å lese og vedlikeholde.